app/assets/javascripts/social_networking/controllers/goal-controller.js

```diff
@@ -154,6 +155,13 @@
     this._goalTool.setFilter(type);
   };
 
+  GoalCtrl.prototype.showCharLimit = function(inputTag) {
```

Comment:
@ericcf I thought about placing this method in a service, but I didn't see a huge advantage.  Please let me know your thoughts.